app/controllers/admin/lead_provider_users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Admin::LeadProviderUsersController < Admin::BaseController
+  skip_after_action :verify_policy_scoped
+  before_action :load_lead_provider, only: %i[edit update]
+  before_action :load_lead_provider_user, only: %i[edit update]
```

Comment:
Do we need `only`s? Are there any other actions here?

Reply:
@kicferk1 What's 'only s'

Reply:
```
only: %i[edit update]
```
in each before_action

Reply:
Got you

Reply:
There will be more actions coming but be can add the restriction when those actions have been added

Reply:
If what's coming is 'show', I would imagine we can remove it. Cause surely show will need it.

If what's coming is 'index', we can keep it. Cause index won't need it.

If something else, uh, I will let you choose :)